calico_containers/pycalico/netns.py

```diff
@@ -51,6 +51,20 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    Increment the metric of any default route that has a metric of 0.
+    :param namespace: The Networking namespace of the container.
+    :return: None. Raises CalledProcessError on error.
+    """
+    with NamedNamespace(namespace) as ns:
+        routes = ns.check_output(["ip", "route"]).split("\n")
+        for route in routes:
+            if route.startswith("default") and "metric" not in route:
+                ns.check_output(["ip", "route", "del"] + route.split())
```

Comment:
I toyed a lot with using `ip route change` to modify the metric, but couldn't get it to accept it:
```
/ # ip route change default via 172.17.42.1 dev eth0 metric 1
ip: RTNETLINK answers: Operation not permitted
```

Note, this was on busybox, which almost seemed like it did not support `ip route change` as a function.